gma/src/include/firebase/gma.h

```diff
@@ -55,15 +55,32 @@ template <typename ResultType>
 class Future {
   // Empty class (used for documentation only).
 };
+
```

Comment:
This is needed because the newly created GMA deprecation page throws a docs warning since it can't find firebase::App docs.